src/schema/v2/artwork/index.ts

```diff
@@ -223,6 +223,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      isEdition: {
+        type: GraphQLBoolean,
+        resolve: ({ edition_sets }) => {
+          return edition_sets && edition_sets.length >= 1
```

Comment:
Would love some eyes on this to make sure this is valid logic 👀

Reply:
Looks OK to me, but there may be hidden weirdness I don't know about?